gameplay/src/Ref.h

```diff
@@ -43,6 +43,41 @@ class Ref
      */
     unsigned int getRefCount() const;
 
+    /**
```

Comment:
We don't plan to implement this as a smart pointer. Ref class is used explicitly.